build.gradle

```diff
@@ -43,6 +43,7 @@ task detektAll(type: io.gitlab.arturbosch.detekt.Detekt) {
     include("**/*.kts")
     exclude("**/build/**")
     exclude("**/test/**/*.kt")
+    exclude("**/testDefaults/**/*.kt")
```

Comment:
Honestly, I would like to run the linter in the tests as well... But there were a bunch of issues and didn't want to make this PR even bigger, so I'm just excluding this for now.